Libs/vtkSegmentationCore/vtkSegmentationModifier.cxx

```diff
@@ -337,16 +359,19 @@ bool vtkSegmentationModifier::AppendLabelmapToSegment(vtkOrientedImageData* labe
 
     if (operation == vtkOrientedImageDataResample::OPERATION_MINIMUM && !minimumOfAllSegments)
       {
+      int extentUnion[6];
+      vtkSegmentationModifier::GetExtentUnion(resampledSegmentLabelmap, extent, extentUnion);
```

Comment:
@Sunderlandkyl could you check this? Is `extent` specified in `resampledSegmentLabelmap` geometry?

It would be nice if you could suggest documentation for vtkSegmentationModifier::AppendLabelmapToSegment that clarifies which input `extent` is defined for and what is resampled to what.

And please if you can have a look at the changes overall and tell us if they look good to you. Thank you!

Reply:
Extent seems to be the extent in labelmap geometry. I'll add documentation/renaming + make the meaning consistent.